src/main/native/cpp/view/Analyzer.cpp

```diff
@@ -84,7 +85,7 @@ void Analyzer::Display() {
     ImGui::SetNextItemWidth(ImGui::GetFontSize() * 15);
     if (ImGui::Combo("Dataset", &m_settings.dataset, AnalysisManager::kDatasets,
                      m_type == analysis::kDrivetrain ? 9 : 3)) {
-      Calculate();
+      Calculate();  // TODO: Determine if Plot Method needed here.
```

Comment:
Why the TODO?

Reply:
The plots didn't seem to be refreshed when the data sets are changed. I wasn't sure if there was a reason for this.